pkg/apis/nais.io/v1/digdirator_types.go

```diff
@@ -263,13 +255,7 @@ type IDPortenClientSpec struct {
 	SecretName string `json:"secretName"`
 	// Register different oauth2 Scopes on your client.
 	// You will not be able to add a scope to your client that conflicts with the client's IntegrationType.
-	// For example, you can not add a scope that is limited to the IntegrationType `krr` of integrationType `idporten`, and vice versa.
-	//
-	// Default for IntegrationType `krr` = ("krr:global/kontaktinformasjon.read", "krr:global/digitalpost.read")
-	// Default for IntegrationType `idporten` = ("openid", "profile")
-	// IntegrationType `api_klient` have no Default, checkout Digdir documentation.
-	//
-	// +nais:doc:Link="https://docs.digdir.no/oidc_func_clientreg.html?h=api_klient#scopes"
```

Comment:
Why was this removed? The documentation is still useful in the context of the `IDPortenClient` CRD.

Reply:
adding :)

Reply:
Still missing the rest of the text for scopes. The `kubebuilder` validation tags for `IntegrationType` has now also disappeared. I think the text before was fine, no need to change it 😄